plugins/hls-call-hierarchy-plugin/README.md

```diff
@@ -0,0 +1,27 @@
+# Call hierarchy plugin for the [Haskell Language Server](https://github.com/haskell/haskell-language-server#readme)
+
+The call hierarchy plugin can review the code to determine where functions are called and how they relate to other functions.
+
+This plugin is useful when debugging and refactoring code because it allows you to see how different parts of the code are related. And it is more conducive for users to quickly understand their macro architecture in the face of strange code.
+
+## Demo
+
+![Call Hierarchy in Emacs](call-hierarchy-in-emacs.gif)
+
+![Call Hierarchy in VSCode](call-hierarchy-in-vscode.gif)
+
+## Prerequisite
+None. You can experience the whole feature without any setting.
+
+## Configuration
+Enable default. You can disable it in the setting whenever you like.
```

Comment:
```suggestion
Enabled by default. You can disable it in your editor settings whenever you like.

```json
{
  "haskell.plugin.callHierarchy.globalOn": true
}
```
```

Reply:
@July541 have you checked the hls commands related with the config include hierarchy (`haskell-language-server vscode-extension-schema` and `haskell-language-server generate-default-config` ? I think the generic config option to enable/disable globally it is `haskell.plugin.callHierarchy.globalOn` but not sure at 100%

Reply:
I tested the two commands, it's really like you said!